src/patternfly/components/Button/styles.scss

```diff
@@ -135,21 +147,29 @@
   &.pf-m-hover {
```

Comment:
does it not? I have the `:active` on `&:hover, &.pf-m-hover`

Reply:
the preview doesnt seem to react on hover

Reply:
I think the issue might be that the colors are so similar.

The default is `$pf-color-blue-400` and hover is `$pf-color-blue-500`, so it can be difficult to notice the difference (at least when I looked at it locally).

Reply:
The problem is not the color different, it's the size of the border. when I hover over the secondary btn, it doesn't grow to 2px

Reply:
ah, I see - should be fixed now!